packages/zoe/src/contractFacet/types.ts

```diff
@@ -13,10 +13,17 @@ import type { Subscriber } from '@agoric/notifier';
 import type {
   AmountKeywordRecord,
   ExitRule,
+  FeeMintAccess,
+  Instance,
   InvitationDetails,
+  ProposalRecord,
+  UserSeat,
   ZoeService,
 } from '../types-index.js';
 
+// XXX runtime import of types
+import './internal-types.js';
```

Comment:
how could this have runtime impact? it's a .ts file; don't we provide a companion `.js` file that's empty?

Reply:
good point. I should say "side-effect" import is more accurate.